lib/engines/content_object_store/features/step_definitions/content_object_store_steps.rb

```diff
@@ -37,7 +37,8 @@
 When("I click on the {string} schema") do |schema_id|
   @schema = @schemas[schema_id]
   ContentObjectStore::ContentBlock::Schema.expects(:find_by_block_type).with(schema_id).at_least_once.returns(@schema)
-  click_link @schema.name
+  choose @schema.name
+  click_on "Save and continue"
```

Comment:
I've noticed we have a mix of where clicking on the submit button some places is separated into a different step, other times is included in a step - very minor little quibble but would be nice to just have it as a separate step where we can I think, as when/if the button text changes we only have to change it in one place